packaging_ignore.yml

```diff
@@ -0,0 +1,23 @@
+base:
+  - ".* warning: ignoring old recipe for target 'check'"
+  - "sh: warning: setlocale: LC_ALL: cannot change locale \\(C.utf8\\): No such file or directory"
+  - "dpkg-buildpackage: warning: using a gain-root-command while being root"
+  - "/sbin/ldconfig: Warning: ignoring configuration file that cannot be opened: /etc/ld.so.conf: No such file or directory"
+  - "warning: line \\d+: multiple %files for package *"
+  - "configure: WARNING: unrecognized options: --disable-dependency-tracking"
+  - "WARNING: If you run into issues during linking or bitcode compilation, you can use --without-security-flags."
+  - "gpg: WARNING: options in `/root/.gnupg/gpg.conf' are not yet active during this run"
+  - ".* warning: overriding recipe for target 'check'"
+  - "dpkg-genchanges: warning: the current version .* is earlier than the previous one .*"
+  - ".*: warning: '__s' may be used uninitialized in this function \\[-Wmaybe-uninitialized\\].*"
+  - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
+  - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
+  - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
+
+debian:
+  - ".*: W: invalid-license Commercial"
+  - ".*: W: no-documentation"
```

Comment:
Do we want to keep Debian specific rules here as well?

I am ok with keeping those here for consistency. There is no harm in doing this I think.

Reply:
I just want it consistent as much as possible as you mentioned